dashboard/src/containers/LayoutContainer/LayoutContainer.ts

```diff
@@ -0,0 +1,12 @@
+import { connect } from "react-redux";
+
+import Layout from "../../components/Layout";
+import { IStoreState } from "../../shared/types";
+
+function mapStateToProps({ config }: IStoreState) {
+  return {
+    kubeappsVersion: config.kubeappsVersion,
```

Comment:
If this is being used only in the Footer, why didn't we create a FooterContainer instead?

Reply:
That way you do not need to pass down the props through the layout component.

Reply:
I do not think I fully understand the reasoning. Generally speaking, do you mean that you want to avoid importing a `container` from a `component`? Why is that?